src/all.rs

```diff
@@ -0,0 +1,335 @@
+    &MEROITIC_HIEROGLYPHS,
+    &MIAO,
+    &MISCELLANEOUS_MATHEMATICAL_SYMBOLS_A,
+    &MISCELLANEOUS_MATHEMATICAL_SYMBOLS_B,
+    &MISCELLANEOUS_SYMBOLS,
+    &MISCELLANEOUS_SYMBOLS_AND_ARROWS,
+    &MISCELLANEOUS_SYMBOLS_AND_PICTOGRAPHS,
+    &MISCELLANEOUS_TECHNICAL,
+    &MODI,
+    &MODIFIER_TONE_LETTERS,
+    &MONGOLIAN,
+    &MONGOLIAN_SUPPLEMENT,
+    &MRO,
+    &MULTANI,
+    &MUSICAL_SYMBOLS,
+    &MYANMAR,
+    &MYANMAR_EXTENDED_A,
+    &MYANMAR_EXTENDED_B,
+    &NABATAEAN,
+    &NAG_MUNDARI,
+    &NANDINAGARI,
+    &NEWA,
+    &NEW_TAI_LUE,
+    &NKO,
+    &NUMBER_FORMS,
+    &NUSHU,
+    &NYIAKENG_PUACHUE_HMONG,
+    &OGHAM,
+    &OLD_HUNGARIAN,
+    &OLD_ITALIC,
+    &OLD_NORTH_ARABIAN,
+    &OLD_PERMIC,
+    &OLD_PERSIAN,
+    &OLD_SOGDIAN,
+    &OLD_SOUTH_ARABIAN,
+    &OLD_TURKIC,
+    &OLD_UYGHUR,
+    &OL_CHIKI,
+    &OPTICAL_CHARACTER_RECOGNITION,
+    &ORIYA,
+    &ORNAMENTAL_DINGBATS,
+    &OSAGE,
+    &OSMANYA,
+    &OTTOMAN_SIYAQ_NUMBERS,
+    &PAHAWH_HMONG,
+    &PALMYRENE,
+    &PAU_CIN_HAU,
+    &PHAGS_PA,
+    &PHAISTOS_DISC,
+    &PHOENICIAN,
+    &PHONETIC_EXTENSIONS,
+    &PHONETIC_EXTENSIONS_SUPPLEMENT,
+    &PLAYING_CARDS,
+    &PRIVATE_USE_AREA,
+    &PSALTER_PAHLAVI,
+    &REJANG,
+    &RUMI_NUMERAL_SYMBOLS,
+    &RUNIC,
+    &SAMARITAN,
+    &SAURASHTRA,
+    &SHARADA,
+    &SHAVIAN,
+    &SHORTHAND_FORMAT_CONTROLS,
+    &SIDDHAM,
+    &SINHALA,
+    &SINHALA_ARCHAIC_NUMBERS,
+    &SMALL_FORM_VARIANTS,
+    &SMALL_KANA_EXTENSION,
+    &SOGDIAN,
+    &SORA_SOMPENG,
+    &SOYOMBO,
+    &SPACING_MODIFIER_LETTERS,
+    &SPECIALS,
+    &SUNDANESE,
+    &SUNDANESE_SUPPLEMENT,
+    &SUPERSCRIPTS_AND_SUBSCRIPTS,
+    &SUPPLEMENTAL_ARROWS_A,
+    &SUPPLEMENTAL_ARROWS_B,
+    &SUPPLEMENTAL_ARROWS_C,
+    &SUPPLEMENTAL_MATHEMATICAL_OPERATORS,
+    &SUPPLEMENTAL_PUNCTUATION,
+    &SUPPLEMENTAL_SYMBOLS_AND_PICTOGRAPHS,
+    &SUPPLEMENTARY_PRIVATE_USE_AREA_A,
+    &SUPPLEMENTARY_PRIVATE_USE_AREA_B,
+    &SUTTON_SIGNWRITING,
+    &SYLOTI_NAGRI,
+    &SYMBOLS_AND_PICTOGRAPHS_EXTENDED_A,
+    &SYMBOLS_FOR_LEGACY_COMPUTING,
+    &SYRIAC,
+    &SYRIAC_SUPPLEMENT,
+    &TAGALOG,
+    &TAGBANWA,
+    &TAGS,
+    &TAI_LE,
+    &TAI_THAM,
+    &TAI_VIET,
+    &TAI_XUAN_JING_SYMBOLS,
+    &TAKRI,
+    &TAMIL,
+    &TAMIL_SUPPLEMENT,
+    &TANGSA,
+    &TANGUT,
+    &TANGUT_COMPONENTS,
+    &TANGUT_SUPPLEMENT,
+    &TELUGU,
+    &THAANA,
+    &THAI,
+    &TIBETAN,
+    &TIFINAGH,
+    &TIRHUTA,
+    &TOTO,
+    &TRANSPORT_AND_MAP_SYMBOLS,
+    &UGARITIC,
+    &UNIFIED_CANADIAN_ABORIGINAL_SYLLABICS,
+    &UNIFIED_CANADIAN_ABORIGINAL_SYLLABICS_EXTENDED,
+    &UNIFIED_CANADIAN_ABORIGINAL_SYLLABICS_EXTENDED_A,
+    &VAI,
+    &VARIATION_SELECTORS,
+    &VARIATION_SELECTORS_SUPPLEMENT,
+    &VEDIC_EXTENSIONS,
+    &VERTICAL_FORMS,
+    &VITHKUQI,
+    &WANCHO,
+    &WARANG_CITI,
+    &YEZIDI,
+    &YIJING_HEXAGRAM_SYMBOLS,
+    &YI_RADICALS,
+    &YI_SYLLABLES,
+    &ZANABAZAR_SQUARE,
+    &ZNAMENNY_MUSICAL_NOTATION,
+];
```

Comment:
Why not order by the block range?

By the way, I generate `UnicodeBlock` by using another Rust program which I haven't open-sourced. If we need an array of `UnicodeBlock`s, this array should also be generated by it.

Reply:
> Why not order by the block range? 

I guess the order depends on the application. For my case it didn't really matter. So sure, the refs could as well be ordered by block range.

> By the way, I generate UnicodeBlock by using another Rust program which I haven't open-sourced. If we need an array of UnicodeBlocks, this array should also be generated by it.

I agree. It would be nice to include your generator in this crate so people submitting PRs like mine could extend the generator.